src/utils/routes/PrivateRoute.tsx

```diff
@@ -1,14 +1,23 @@
 import React, { FC, ComponentType } from 'react';
 import { Route, RouteProps } from 'react-router-dom';
 import Forbidden from '../../pages/errorpages/Forbidden';
+import { useSelector } from 'react-redux';
+import { RootStore } from '../../redux/reducers/rootReducer';
 
 export interface PrivateRouteProps extends RouteProps {
   component: ComponentType<any>;
   isAuthorized: boolean;
 }
 
-const PrivateRoute: FC<PrivateRouteProps> = ({ component: Component, isAuthorized, ...rest }) => (
-  <Route {...rest} render={(props) => (isAuthorized ? <Component {...props} {...rest} /> : <Forbidden />)} />
-);
+const PrivateRoute: FC<PrivateRouteProps> = ({ component: Component, isAuthorized, ...rest }) => {
+  const user = useSelector((store: RootStore) => store.user);
+
+  return (
+    <Route
+      {...rest}
+      render={(props) => (isAuthorized ? <Component {...props} {...rest} /> : !!user ? <Forbidden /> : null)}
+    />
+  );
+};
 
```

Comment:
Burde man ikke få opp Forbidden om man går til /user uten å være innnlogget også?
![bilde](https://user-images.githubusercontent.com/6313468/86211954-7c5a0c80-bb77-11ea-8a65-0dad8f02a184.png)

Reply:
tanken var egentlig at vi viser spinner helt til user er lastet (App.tsx)

Reply:
Jo, vi bør jo egentlig det. Kan ta denne tilbake igjen for å finne en annen måte å ikke flashe forbidden page etter at man har logget ut